contributed_definitions/nyaml/NXxps.yaml

```diff
@@ -0,0 +1,474 @@
+category: application
+doc: |
+  This is the application definition for X-ray photoelectron spectroscopy.
+type: group
+NXxps(NXmpes):
+  (NXentry):
+    definition(NX_CHAR):
+      enumeration: [NXxps]
+    method:
+      doc: |
+        A name of the experimental method according to `Clause 11`_ of
+        the `ISO 18115-1:2023`_ specification.
+        
+        Examples for XPS-related experiments include:
+            * X-ray photoelectron spectroscopy (XPS)
+            * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
+            * ultraviolet photoelectron spectroscopy (UPS)
+            * hard X-ray photoemission spectroscopy (HAXPES)
+            * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
+            * electron spectroscopy for chemical analysis (ESCA)
+        
+        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+        .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
+    geometries(NXcoordinate_system_set):
+      xps_coordinate_system(NXcoordinate_system):
+        exists: optional
+        doc: |
+          In traditional surface science, the coordinate system is defined such that the positive z-axis
+          points along the normal of the sample stage, and the x- and y-axes lie in the plane of the sample stage.
+          However, in NeXus, a coordinate system that is the same as `McStas`_  is used. `xps_coordinate_system`
+          gives the user the opportunity to work in the traditional base coordinate system.
+          
+          .. _McStas: http://mcstas.org/
+          
+          .. image:: xps/xps_cs.png
+              :width: 40%
+        origin:
+          enumeration: [sample stage]
+        handedness:
+          enumeration: [left_handed]
```

Comment:
That seems non-standard. Why?

Reply:
I am building this coordinate system based on the one defined in the ISO standard. Unfortunately, I cannot share this here (the standard is propietary), but the CS defined there is left handed.